Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -405,37 +405,9 @@ namespace webrtc
             RTC_LOG(LS_ERROR) << "cannot get export handle";
             return nullptr;
         }
-
```

Comment:
I think the handle is still created here but not used anymore? I was creating the handle early and using it against the cu* calls directly, so that in case there are any errors and part of the initialized stuff has to be withdraw, the destructor of the GpuMemoryBufferCudaHandle handles that.

Reply:
Sorry, I failed to rebase to the main branch. I am fixing it.

Reply:
:) Sorry for nitpicking, there's two cases left where GpuMemoryBufferCudaHandle gets created late vs original code

Reply:
Could you tell me where there's two cases?

Reply:
Sorry but I might misunderstood your advice. Can you see me the patch?

Reply:
Thanks, cherry-picked it.